scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -8,6 +8,7 @@
 import argparse
 import contextlib
 import glob
+from io import StringIO
```

Comment:
flake8 complains
./scripts/jenkins/ardana/gerrit/build_test_package.py:12:1: I100 Import statements are in the wrong order. 'import os' should be before 'from io import StringIO'

Reply:
Yes, I pushed the wrong commit.